<ui version="4.0" >
 <class>SqlMakerWidgetClass</class>
 <widget class="QWidget" name="SqlMakerWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sql Maker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Table : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QComboBox" name="tableCmb" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tableDescLbl" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout_4" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Fields : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fieldCntLbl" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="arrangeBtn" >
       <property name="text" >
        <string>Arrange</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QTextEdit" name="fieldEdit" />
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Generated</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QRadioButton" name="normalRdo" >
          <property name="text" >
           <string>Normal</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bindRdo" >
          <property name="text" >
           <string>Bind Values</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>178</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="1" >
       <widget class="QTextEdit" name="insertEdit" />
      </item>
      <item row="4" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Update : </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="1" >
       <widget class="QTextEdit" name="updateEdit" />
      </item>
      <item row="5" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Delete : </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="1" >
       <widget class="QTextEdit" name="deleteEdit" />
      </item>
      <item row="3" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Insert : </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1" >
       <widget class="QTextEdit" name="selectEdit" />
      </item>
      <item row="2" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout_5" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Select : </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
